app/models/organization.rb

```diff
@@ -94,6 +108,10 @@ def generate_api_key
     self.api_key = SecureRandom.uuid
   end
 
+  def generate_document_number_prefix
+    update!(document_number_prefix: "#{name.first(3).upcase}-#{id.last(4).upcase}")
```

Comment:
small comment to precise what the format ?

Reply:
Added